### DISCUSSION TITLE
Time to live for Server-side Storage

### DISCUSSION THREAD

**Discussion by JS-Aibel** [zauberzeug/nicegui#1397] [Q&A] [answered]
### Question

Is it possible to set the app.storage.user data on a time to live?

**Use case**
I want the UI state to be stored in case of a connection drop, but do not want to fill up the server with unused data.  Is it possible to set the app.storage.user data to a TTL of for example 12 hours per user?

Ref. [storage • NiceGUI](https://nicegui.io/documentation/storage)

**Comment by falkoschindler** [comment, marked as answer]
I think this is similar to feature request https://github.com/zauberzeug/nicegui/discussions/991.

**Reply by JS-Aibel** [reply]
Thank you, sorry for double posting.
